packages/attestation-service/src/requestHandlers/get_attestation.ts

```diff
@@ -87,10 +96,35 @@ class GetAttestationRequestHandler {
 }
 
 export async function handleGetAttestationRequest(
-  _req: express.Request,
+  req: express.Request,
   res: express.Response,
   getRequest: GetAttestationRequest
 ) {
+  const {
+    headers: { authentication },
+  } = req
+  if (!authentication) {
+    respondWithError(res, 401, 'Missing authentication')
+    return
+  }
+
+  if (getRequest.securityCode) {
+    const typedData = buildSecurityCodeTypedData(getRequest.securityCode)
+    const accounts = await useKit((kit) => kit.contracts.getAccounts())
+    const [walletAddress, dekAddress] = await Promise.all([
+      accounts.getWalletAddress(getRequest.account),
+      accounts.getDataEncryptionKey(getRequest.account).then((x) => x && publicKeyToAddress(x)),
+    ])
+
+    const validSignature = [getRequest.account, walletAddress, dekAddress]
```

Comment:
oh interesting, does MTW sign the code as the account or with the DEK? seems this implementation would support either